### ISSUE TITLE
Cronjobs header in URL aufrufen hinzufügen

### ISSUE THREAD

**Issue by leonschwanitz** [mittwald/feature-requests#102]
**Welches Problem möchtest du lösen? Wann tritt es auf?**
Beim Erstellen von Cronjobs möchte ich einen Header angeben können, zum Beispiel einen Cron-Token, um den Cronjob im Backend zu validieren.

**Welche Lösungsideen hast du?**
Man kann mehrere Header über das Interface hinzufügen.

**Comment by patrickhilker** [comment]
Über Umwege wäre das übrigens schon jetzt möglich: Einfach ein kleines Bash-Script anlegen, das per `cURL` den HTTP-Request inkl. Header macht. Das dann per Cronjob aufrufen.

Aber schöner wäre natürlich, wenns bequem per mStudio ginge. :-)

**Comment by leonschwanitz** [comment]
@patrickhilker Genau! :) Schöner wäre es wenn man es direkt übers Interface machen könnte. Auch direkt mit POST request.
